eq-author/src/components/modals/ModalWithNav/ModalWithNav.test.js

```diff
@@ -55,37 +55,57 @@ const NAV_ITEMS = [
 ];
 
 describe("ModalWithNav", () => {
-  let wrapper, props;
+  let props;
 
   beforeEach(() => {
     props = {
-      id: "TEST_MODAL",
       title: "I am a title",
       onClose: jest.fn(),
-      gotoTab: jest.fn(),
+      startingTabId: "tab-1",
+      isOpen: true,
       navItems: NAV_ITEMS,
     };
   });
 
+  it("should not render anything when isOpen is false", () => {
+    const { queryByText } = render(<ModalWithNav {...props} isOpen={false} />);
+    expect(queryByText("Tellus Dollor")).toBeFalsy();
```

Comment:
I think this will be `falsy` even if it is open (misspelled `Dolor`). And maybe add a check to see that it is `truthy` when it is open.